### ISSUE TITLE
super-slim process list for  "big gene list tool"

### ISSUE THREAD

**Issue by ValWood** [pombase/website#886]
we'll begin with this 'superslim for processes'
Because they need to be mutually exclusive the higher position in the list trumps 
(I will probably refine  but this gives the most coverage right now (all but  182 genes with processes)

signaling (GO:0023052) >
gene expression (GO:0010467) >
chromatin remodeling (GO:0006338)>
protein folding (GO:0006457)>
cellular component biogenesis (GO:0044085) >
DNA metabolic process (GO:0006259)>
cell cycle (GO:0007049)	>
cytoskeleton organization (GO:0007010) >
membrane organization (GO:0061024) >
organelle localization (GO:0051640) >
lipid metabolic process (GO:0006629) >
small molecule metabolic process (GO:0044281) >
generation of precursor metabolites and energy (GO:0006091) >
transport (GO:0006810) >
cellular catabolic process (GO:0044248)	>
detoxification (GO:0098754)

**Comment by kimrutherford** [comment]
I've added this now (I think).  I've implemented it in the same way as the component list - the terms at the top of your list have the highest priority.  I generated a random colour palette and some of them are a bit nasty.

The configuration is in `pombe-embl/website/pombase_v2_config.json`.  Look for this section:

```json
  "query_data_config": {
    "go_components": [
      "GO:0005634", "GO:0005783", "GO:0005739", "GO:0005737"
    ],
    "go_process_superslim": [
      "GO:0023052", "GO:0010467", "GO:0006338", "GO:0006457", "GO:0044085",
      "GO:0006259", "GO:0007049", "GO:0007010", "GO:0061024", "GO:0051640",
      "GO:0006629", "GO:0044281", "GO:0006091", "GO:0006810", "GO:0044248",
      "GO:0098754"
    ],
    "ortholog_presence_taxonids": [9606, 4932]
  },
```

If you make edits the change will appear the next day.  If you let me know when you commit changes, I'll look out for problems.

It's currently called "go_process_superslim" internally because I thought that sounds cool.

**Comment by ValWood** [comment]
Only just got to this ticket! I'm guessing this went away with the rollback. Looking forward to test driving it soon.

**Comment by kimrutherford** [comment]
> Only just got to this ticket! I'm guessing this went away with the rollback. Looking forward to test driving it soon.

Yep, I rolled back pombase-dev too since it wasn't working.  It's working now.

**Comment by ValWood** [comment]
Nice. W can discuss colour in final stage, but I don't think colours between bars necessarily need to be unique which will give more choices. can discuss.
